apps/prairielearn/src/pages/authCallbackShib/authCallbackShib.ts

```diff
@@ -1,10 +1,9 @@
-// @ts-check
 import { Router } from 'express';
 import asyncHandler from 'express-async-handler';
 
-import * as error from '@prairielearn/error';
+import { HttpStatusError } from '@prairielearn/error';
 
-import * as authnLib from '../../lib/authn.js';
+import { loadUser } from '../../lib/authn.js';
```

Comment:
@trombonekenny previously expressed a preference to keep `import * as ...` so that the call is `authnLib.loadUser`. I'm not sure if he still wants that or not. Regardless, let's be consistent between this, `authCallbackOauth2`, and others.

Reply:
Restored in ac1432abc.